Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -934,9 +932,32 @@ bool Renderer::StartFrameDumpToImage(const FrameDumpConfig& config)
   return true;
 }
 
+void writePPM(std::ostream& os, int width, int height, int stride, const u8* data)
+{
+  using namespace std;
```

Comment:
There's not really a need to use it here. The exception of the rule is when you want to force argument-dependent lookup when specializing a function (e.g. std::swap specializations).